docs/roadmap.rst

```diff
@@ -45,14 +34,11 @@ KPIs
 Features
 ++++++++
 
--   process overview can be filtered by region/city
 -   idea collection process with or without a map
 -   private and public processes
--   project managers can invite new users
+-   new users can be invited by liqd
 -   users can edit their own account information
--   project managers can generate embed snippets for a process
--   moderators of processes can download a data file (e.g. CSV) with
-    process contents
+-   embed snippets can be generated for a process
```

Comment:
This is already possible. The point is to simplify it to a point where project managers can do it independently.